server/utils/utils.ts

```diff
@@ -38,6 +39,33 @@ const hasAuthSource = (user: Express.User, source: string): boolean => user?.aut
  */
 const convertDateFormat = (date: string): string => (date ? moment(date, 'YYYY-MM-DD').format('DD/MM/YYYY') : undefined)
 
+/**
+ * Converts a JSON date time `dd/mm/yyyy hh:mm` to a DateTime for display
+ * @param dt: string
+ */
+const jsonToDateTime = (dt: string): DateTime | undefined => {
+  const momentDt = moment(dt, 'D/MM/YYYY HHmm')
+  if (!momentDt.isValid()) {
+    return undefined
+  }
+  const dateString = momentDt.format('DD/MM/YYYY')
+  const ampm = momentDt.format('a') === 'am' ? AmPm.AM : AmPm.PM
+  const simpleTime = new SimpleTime(momentDt.format('hh'), momentDt.format('mm'), ampm)
+  return DateTime.fromDateAndTime(dateString, simpleTime)
+}
+
+/**
+ * Converts a DateTime display value to a JSON string format dd/mm/yyyy hh:mm
+ * @param dt: DateTime
+ */
+const dateTimeToJson = (dt: DateTime): string | undefined => {
```

Comment:
would this be better as a function on DateTime?

Reply:
sortd in a follow up PR